kafkarest/v3/api_cluster_v3.go

```diff
@@ -42,59 +42,58 @@ var (
 type ClusterV3Api interface {
 
 	/*
-	GetKafkaV3Cluster Get Cluster
+			GetKafkaCluster Get Cluster
 
-	[![Generally Available](https://img.shields.io/badge/Lifecycle%20Stage-Generally%20Available-%2345c6e8)](#section/Versioning/API-Lifecycle-Policy)
+			[![Generally Available](https://img.shields.io/badge/Lifecycle%20Stage-Generally%20Available-%2345c6e8)](#section/Versioning/API-Lifecycle-Policy)
 
-Returns the Kafka cluster with the specified ``cluster_id``.
+		Returns the Kafka cluster with the specified ``cluster_id``.
 
-	 @param ctx _context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
-	 @param clusterId The Kafka cluster ID.
-	 @return ApiGetKafkaV3ClusterRequest
+			 @param ctx _context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+			 @param clusterId The Kafka cluster ID.
+			 @return ApiGetKafkaClusterRequest
 	*/
-	GetKafkaV3Cluster(ctx _context.Context, clusterId string) ApiGetKafkaV3ClusterRequest
+	GetKafkaCluster(ctx _context.Context, clusterId string) ApiGetKafkaClusterRequest
 
-	// GetKafkaV3ClusterExecute executes the request
+	// GetKafkaClusterExecute executes the request
 	//  @return ClusterData
-	GetKafkaV3ClusterExecute(r ApiGetKafkaV3ClusterRequest) (ClusterData, *_nethttp.Response, error)
+	GetKafkaClusterExecute(r ApiGetKafkaClusterRequest) (ClusterData, *_nethttp.Response, error)
```

Comment:
Are you ok with losing the V3 part in the function names? Doesn't matter to me

Reply:
That's a great catch! I think the team responsible for this API made this decision on purpose actually.

Reply:
All of the APIs seem to have that same change